packages/@expo/router-server/src/routes-manifest.ts

```diff
@@ -1,8 +1,8 @@
 // This file runs in Node.js environments.
 // no relative imports
-import { type Options, getRoutes } from './getRoutesSSR';
-import { getServerManifest } from './getServerManifest';
-import { type RequireContext } from './types';
+import { type Options, getRoutes } from 'expo-router/build/getRoutesSSR';
+import { getServerManifest } from 'expo-router/build/getServerManifest';
+import { type RequireContext } from 'expo-router/build/types';
```

Comment:
Is it safe for us to import from `build` folder?

Maybe we can have something like `expo-router/internal` for this use-case?